osu.Game/Beatmaps/WorkingBeatmap.cs

```diff
@@ -298,6 +298,7 @@ public void PrepareTrackForPreviewLooping()
         /// Get the loaded audio track instance. <see cref="LoadTrack"/> must have first been called.
         /// This generally happens via MusicController when changing the global beatmap.
         /// </summary>
+        [NotNull]
```

Comment:
What?

This can 99999% be null. Check the code. Unless you are going to actually enforce that it is not null, don't add this attribute.

Reply:
To clarify, it is loosely guarded by an exception of a different variety. I'm not sure this is a valid attribute to add unless you are also going to document the exception.

Reply:
Well, your second comment explains it, will remove then, my bad.